script.js

```diff
@@ -0,0 +1,176 @@
+const cardForm = document.querySelector("#card-form");
+const numberInput = document.querySelector("#card-number-input");
+const number = document.querySelector("#card-number");
+const nameInput = document.querySelector("#card-name-input");
+const cardHolderName = document.querySelector("#name");
+const monthInput = document.querySelector("#month-input");
+const yearInput = document.querySelector("#year-input");
+const expMonth = document.querySelector("#exp-month");
+const expYear = document.querySelector("#exp-year");
+const cvcInput = document.querySelector("#cvc-input");
+const cvc = document.querySelector("#cvc");
+const confirmBtn = document.querySelector("#confirm");
+const continueBtn = document.querySelector("#continue");
+const inputError = document.querySelectorAll(".input-error");
+const inputArr = document.querySelectorAll("input");
+
+let isValid = [false];
+let isEmpty = true;
+
+const format = (num) => {
+  try {
+    let newNum = num.match(/.{1,4}/g).join(" ");
+    return newNum;
+  } catch (e) {
+    console.error("empty number");
+  }
+};
+
+cardForm.addEventListener("submit", (e) => {
+  for (let i = 0; i < inputArr.length; i++) {
+    if (validateNotEmpty(inputArr[i], inputError[i])) {
+      isValid[i] = false;
+      isEmpty = true;
+    } else {
+      validateNotEmpty(inputArr[i], inputError[i]);
+      isEmpty = false;
+    }
+  }
+  isValid.forEach((el) => {
+    console.log(`isValid=${el}`);
+  });
+  if (!isValid.includes(false)) {
+    console.log(`all data valid ${!isValid.includes(false)}`);
```

Comment:
Always remember to remove debugging statements when you push your changes to github.